Makefile

```diff
@@ -113,3 +114,14 @@ gendocs: docs/minikube.md
 docs/minikube.md: $(shell find cmd) $(shell find pkg/minikube/constants) pkg/minikube/cluster/assets.go
 	$(MKGOPATH)
 	cd $(GOPATH)/src/$(REPOPATH) && go run -ldflags="$(K8S_VERSION_LDFLAGS) $(MINIKUBE_LDFLAGS)" -tags gendocs gen_help_text.go
+
+.PHONY: deb
+deb: 
+	@echo -e "Build .deb package in directory: out/.\n  Note:  There must compiled binary at out/minikube to put in the deb \n"
+	mv installers/linux/deb/minikube_* installers/linux/deb/minikube_$(DEB_VERSION) 2>/dev/null || true
```

Comment:
Ah makes sense. I guess my suggestion is mainly to do all temp directory work in out/ to avoid needing the extra .gitignore. Would this work?

cp installers/linux/deb/minikube_* out/installers/linux/deb/minikube_...
chmod 0755 out/...
sed -E ...
mkdir -p ...
cp out/minikbue-linux-amd64 ...
dpkd-deb
mv...